### PR TITLE
Embcess 716 pdf printer api

### PR DESCRIPTION
**Pull request by CodingVelocista**

### CODE REVIEW

File: embc-app/Controllers/ReferralsController.cs [hunk 1]
```diff
@@ -10,14 +11,16 @@
 namespace Gov.Jag.Embc.Public.Controllers
 {
     [Route("api/registrations/{registrationId}/[controller]")]
-    [Authorize]
+    [AllowAnonymous]
```
Comment (ytqsl) [inline embc-app/Controllers/ReferralsController.cs]: should this be merged?

File: embc-app/embc-app.csproj [hunk 2]
```diff
@@ -34,6 +34,8 @@
     <PackageReference Include="Microsoft.AspNetCore.SpaServices.Extensions" Version="2.1.1" />
 
     <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore" Version="2.2.0" />
+
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.1.5" />
```
Comment (ytqsl) [inline embc-app/embc-app.csproj]: What's that package do?

Comment (CodingVelocista) [reply embc-app/embc-app.csproj]: Don't know where it came from, will revert that for sure.

File: embc-app/Controllers/ReferralsController.cs (hunk 1 above)
Comment (CodingVelocista) [reply embc-app/Controllers/ReferralsController.cs]: Changed

File: embc-app/embc-app.csproj (hunk 2 above)
Comment (CodingVelocista) [reply embc-app/embc-app.csproj]: Removed

**Pull request merged**